src/llm_vm/server/main.py

```diff
@@ -21,7 +22,32 @@
 
 app.register_blueprint(routes.bp)
 
-def main_server_entry_point():
+def main_server_entry_point(host = '127.0.0.1', port = 3002):
     # make this more configurable soon
-    app.run(host="127.0.0.1", port=3002)
-
+    app.run(host = host,port = port)
+
+
```

Comment:
We will want something more modular soon, but for now for input validation this seems ok.

Reply:
Should cli be the new actual entry point?